.github/workflows/linux-build-and-test.yaml

```diff
@@ -96,6 +96,10 @@ jobs:
         make -C nossl install
         make -C nossl regresscheck TESTS=telemetry
 
+    - name: Fetch all tags
+      # We need the tags to be able to build a downgrade script.
+      run: git fetch --tags && git tag
+
```

Comment:
i guess this is no longer needed with the changed default

Reply:
Removed.